### DISCUSSION TITLE
Initial actuator value - slave to input/custom?

### DISCUSSION THREAD

**Discussion by Octal450** [JSBSim-Team/jsbsim#513] [Ideas]
Hi,
I've had this idea for a bit, I currently have a hydraulic breaking system modelled in JSBsim which when the simulation starts, I would like the system to be pressurized as if the aircraft was left in such a state from the last flight. I am using actuators to control the pressure hysteresis and such.

I would propose having a way to make the actuator initialize at the value of its \<input\>, or even have a user-specifiable value (like the PID initial integrator.

In the current implementation, even if I feed the actuator 3000psi for example as the simulation starts, the value will slowly rise up to 3000 at the specified rate-limit - which I want to bypass as I'd like for the system to start pressurized.

Thoughts?
Josh

**Comment by bcoconni** [comment]
Hi @Octal450 

Is the discussion in issue #295 of any help ?
Alternatively, triggering a trim might help initializing the actuator to your liking as in issue #293 ?
Or may be I'm missing something in your request ?

**Comment by Octal450** [comment]
@bcoconni 
Thanks. I am currently in the hospital without a laptop, but will evaluate this as soon as I am home.

I appreciate you bearing with me, I'll try to get back to you shortly.

Kind Regards,
Josh
